feature-libs/organization/account-summary/styles/components/_account-summary-document.scss

```diff
@@ -0,0 +1,237 @@
+cx-account-summary-document {
+  .cx-account-summary-document {
+    @include media-breakpoint-down(md) {
+      max-width: 100%;
+      padding: 0;
+    }
+  }
+
+  .cx-account-summary-document-table {
+    padding: 1.5rem 0 1.125rem 0;
+    margin-bottom: 0;
+    border-top: 1px solid var(--cx-color-light);
+    border-bottom: 1px solid var(--cx-color-light);
+
+    th {
+      padding: 1.5rem 0 1.125rem 0;
+      text-align: start;
+
+      &:last-child {
+        text-align: end;
+      }
+    }
+
+    td {
+      //width: 25%;
+      padding: 1.625rem 0;
+
+      @include media-breakpoint-up(md) {
+        text-align: start;
+
+        //&:last-child {
+        //  text-align: end;
+        //}
+      }
+
+      @include media-breakpoint-down(md) {
+        width: 100%;
+        display: flex;
+        border-width: 0;
+        padding: 0 1.25rem;
+
+        &:first-child {
+          padding-top: 1.25rem;
+        }
+
+        &:last-child {
+          padding-bottom: 1.25rem;
+        }
+      }
+    }
+  }
+
+  .cx-account-summary-document-header {
+    padding: 40px 0 0 0;
+    color: var(--cx-color-text);
+
+    @include media-breakpoint-down(md) {
+      padding-top: 40px;
+      padding-inline-end: 20px;
+      padding-bottom: 0;
+      padding-inline-start: 20px;
+    }
+  }
+
+  .cx-account-summary-document-thead-mobile {
+    @include media-breakpoint-down(md) {
+      display: none;
+    }
+  }
+
+  .cx-account-summary-document-code {
+    .cx-account-summary-document-value {
+      text-decoration: none;
+    }
+  }
+
+  .cx-account-summary-document-type {
+    .cx-account-summary-document-value {
+      text-decoration: none;
+    }
+  }
+
+  .cx-account-summary-document-attachment {
+    .cx-account-summary-document-value {
+      text-decoration: underline;
+      font-weight: 600;
+      text-align: end;
+      color: #FE5757;
```

Comment:
colors should use variables. That way if the client changes the theme the colors will change with the theme. This one should be `color: var(--cx-color-primary);`